template.rb

```diff
@@ -0,0 +1,83 @@
+# Include govuk-rails-app-template root in source_paths
+source_paths << File.dirname(__FILE__)
+
+run 'bundle install'
+git :init
+git add: "."
+git commit: "-a -m 'Bare Rails application'"
```

Comment:
It's probably worth including a link to this repo in the commit message to given an indication how this was generated.